.github/e2eLabeler.yml

```diff
@@ -30,6 +30,7 @@
   - "openmetadata-ui/**/*[Dd]ata[qQ]uality*"
   - "openmetadata-ui/**/*[Dd]ata[Ii]nsight*"
   - "openmetadata-ui/**/*[Oo]bservability*"
+  - "openmetadata-ui/**/*[Dd]ata[Qq]uality*"
```

Comment:
We have this regexp above any reason for add it again

Reply:
Thanks, I have revert the changes!